lib/services/schedule.js

```diff
@@ -121,7 +117,9 @@ function post(uri, data) {
       { type: 'put', key: targetReference, value: JSON.stringify(referencedData) }
     ];
 
-  return db.batch(ops).return(referencedData);
+  return db.batch(ops).then(function () {
+    log('info', 'scheduled', targetUri, `(${data.at})`);
```

Comment:
this is a unix timestamp that we can easily grab and see exactly when something has been logged